tests/test_nomicbert.py

```diff
@@ -0,0 +1,163 @@
+
+As the 1980s rolled in, the increased sophistication of hardware allowed for more intricate game designs and richly detailed narratives. This was the decade where characters and stories began to take center stage, giving rise to iconic figures like Mario and Link, heroes who would become household names. The Nintendo Entertainment System (NES) ushered in an era of home gaming, wresting control from the arcades and planting it firmly within living rooms. The concept of expanded worlds filled with adventure, such as those presented in "The Legend of Zelda," pushed the boundaries of what a video game could achieve, marrying gameplay with an emergent sense of story and discovery. The tactile nature of the controller, combined with the pixelated yet vibrant visuals on screen, offered an experience that was immersive and personal, making it distinct from any other form of media at the time. 
+
+As the 1990s approached, the introduction of 16-bit consoles, exemplified by the Sega Genesis and the Super Nintendo, marked a significant leap in graphical fidelity and sound design. Games could now convey more complex stories and offer richer visual experiences, leading to a golden age of creativity and innovation. This period saw the birth of genre-defining titles like "Sonic the Hedgehog," which combined breathtaking speed with memorable level design, and "Final Fantasy VI," a game that showcased the potential for video games to deliver epic narratives and emotional depth. The battle between Sega and Nintendo during this era spurred rapid advancements in technology and game design, a competitive spirit that benefited gamers with a plethora of high-quality titles to choose from. The resurgence of arcades was also notable, with fighting games like "Street Fighter II" and "Mortal Kombat" drawing crowds with their intense, competitive gameplay and colorful, often controversial characters. This decade also saw the advent of digital pets and life simulation games like "Tamagotchi".
+"""
+
+
+if __name__ == "__main__":
```

Comment:
Could you rather move this test to `tests/onnxruntime/test_modeling.py`'s `ORTModelForFeatureExtractionIntegrationTest`, for example?